Tests/UnitTests/UnitTests.xcodeproj/project.pbxproj

```diff
@@ -655,7 +655,7 @@
 				CLANG_CXX_LIBRARY = "libc++";
 				CLANG_ENABLE_MODULES = NO;
 				CLANG_ENABLE_OBJC_ARC = YES;
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				CODE_SIGN_IDENTITY = "iPhone Developer";
```

Comment:
For my own info: are these equivalent?
"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer"; and
CODE_SIGN_IDENTITY = "iPhone Developer";
??